src/features/MentorPage/ListCard/ListCard.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { ListCardProps } from './types';
+import ListCardHeader from './ListCardHeader';
+import ListCardLanguages from './ListCardLanguages';
+import ListCardSkills from './ListCardSkills';
+import ListCardStory from './ListCardStory';
+import OpenCardButton from './OpenCardButton';
+import * as cssVariables from '../../../components/variables';
+
+/**
+ * Div to hold mentor cards. Needs a parameter added
+ * That hold the info for cards that need to be showed.
+ */
+
+type Props = {
+  setVisibleCard: (mentorCardData: ListCardProps) => void;
+  mentordata: ListCardProps;
+};
+
+const ListCard = ({ setVisibleCard, mentordata }: Props) => {
+  const age = new Date().getFullYear() - mentordata.mentor.birthYear;
+  return (
+    <ListCardElement>
+      <ListCardHeader
+        name={mentordata.mentor.displayName}
+        age={age}
+        region={mentordata.mentor.region}
+        available={mentordata.isLoggedIn}
+        newMentor={mentordata.isNewMentor}
```

Comment:
I prefer to use isAvailable and isNewMentor

newMentor to me refers to a function

Reply:
I am nitpicky with naming, and it is hard =)